modules/ROOT/pages/_partials/showcase-apps/security.adoc

```diff
@@ -0,0 +1,20 @@
+= Security Services
+
+== {device-security-service}
+
```

Comment:
I'd add a line or two here describing what the user will achieve if the execute the steps below.